pkgs/development/python-modules/loompy/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
```

```suggestion
  buildPythonPackage,
  setuptools,
  ... (other python dependencies)
```
Please don't use `python3Packages` in python-modules.
It can't be built against other python versions, like `python311Packages.loompy`.